src/types/configData.go

```diff
@@ -97,12 +97,14 @@ type ConfigClusterMgmt struct {
 }
 
 type ConfigObservability struct {
```

Comment:
Do not bring this under observability config, bring this under generic config.

Reply:
Changes are made and created new generic config.